src/zentables/zentables.py

```diff
@@ -867,43 +884,55 @@ def _local_suppression(
     Helper function that applies cell suppression to mini_df_n.
     mini_df_n is assumed to be a smaller portion of a larger
     """
+
+    df = mini_df_n.copy()
+    colnames = mini_df_n.columns
+    rownames = mini_df_n.index
+    df.columns = np.arange(len(colnames))
```

Comment:
Ugh this is so gross, but it definitely works!

Reply:
Yep 100% agree!